django-server/server/server/settings.py

```diff
@@ -76,12 +78,23 @@
 WSGI_APPLICATION = 'server.wsgi.application'
 
 
+# Configuration for rest_framework to get only the json data
+# from the api calls to specified urls as default
+
+REST_FRAMEWORK = {
+    'DEFAULT_RENDERER_CLASSES': [
+        'rest_framework.renderers.JSONRenderer',
+        # Uncomment to get the API renderer for browser view
+        # 'rest_framework.renderers.BrowsableAPIRenderer',
+    ]
+}
+
+
 # Database
 # https://docs.djangoproject.com/en/3.0/ref/settings/#databases
 
-
 def select_database():
-    if env.bool('USE_POSTGRES', default=False):
+    if env.bool('DB_DEPLOY', default=False):
```

Comment:
We can still deploy to PostgreSQL locally, right?

Reply:
yeah, just define env DB_DEPLOY=True and the env with database auth